backend/src/packages/journal-entries/journal-entry.controller.ts

```diff
@@ -173,6 +176,12 @@ class JournalEntryController extends BaseController {
    *              example:
    *                message: "User with these credentials was not found."
    *                errorType: "USERS"
+   *        422:
+   *          description: Validation failed
+   *          content:
+   *            application/json:
+   *              schema:
+   *                $ref: '#/components/schemas/Unprocessable Journal Entity'
```

Comment:
```suggestion
   *        422:
   *          description: Validation failed
   *          content:
   *            application/json:
   *              schema:
   *                $ref: '#/components/schemas/Unprocessable Journal Entity'
```
So here it is impossible to have this error too